### DISCUSSION TITLE
How to process an incoming media group in Telegram API

### DISCUSSION THREAD

**Discussion by SoulARC** [python-telegram-bot/python-telegram-bot#3561] [Q&A]
The task seems to be simple, I have to accept a media group (photos/videos/files...) in the channel and forward it to another group, but Telegram will send each file separately. It is not clear where the end of this media group is, in addition, it is not clear how to catch them and gather them into 1 media group. Maybe someone faced such a problem?

**Comment by Poolitzer** [comment]
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-requested-design-patterns#how-do-i-deal-with-a-media-group
